statistics/most_attended_competitions_in_single_week.rb

```diff
@@ -0,0 +1,41 @@
+require_relative "../core/statistic"
+
+class MostAttendedCompetitionsInSingleWeek < Statistic
+  def initialize
+    @title = "Most attended competitions in a single week"
+    @table_header = { "Competitions" => :right, "Person" => :left, "&nbsp;Start&nbsp;date&nbsp;" => :left, "&nbsp;End&nbsp;date&nbsp;&nbsp;&nbsp;" => :left, "Year" => :left, "List" => :left }
```

Comment:
```suggestion
    @table_header = { "Competitions" => :right, "Person" => :left, "Week start" => :left, "Week end" => :left, "Year" => :left, "List" => :left }
```

Reply:
That's how it was in the beginning but it looks too ugly imo